bftengine/src/bftengine/SimpleClientImp.cpp

```diff
@@ -446,11 +453,11 @@ uint64_t SeqNumberGeneratorForClientRequestsImp::generateUniqueSequenceNumberFor
       LOG_WARN(GL, "Client SeqNum Counter reached max value");
```

Comment:
can you extract the `0x3FFFFF` value as a constant, also adding a comment that explains its purpose?